Commands/readme.md

```diff
@@ -230,14 +224,11 @@
 - [mobile-network](/Commands/mobile-network/readme.md)
 : Manage mobile network
 
-- [mongo-db](/Commands/mongo-db/readme.md)
-: Manage MongoDB
-
 - [monitor](/Commands/monitor/readme.md)
 : Manage the Azure Monitor Service.
 
 - [neon](/Commands/neon/readme.md)
-: Manage Neon Postgres databases and related resources within Azure.
+: Manage Neon Postgres
```

Comment:
please revert the changes of readme.md. I think it may be caused by using an old version of aaz-dev.